one_fm/setup.py

```diff
@@ -97,6 +109,11 @@ def remove_property_setter(property_setters):
 
 def delete_workflows():
 	delete_workflow(get_workflow_json_file("erf.json"))
+	delete_workflow(get_workflow_json_file("shift_request.json"))
```

Comment:
Use this https://docs.google.com/document/d/1Yef_G0x8YSQOwv2y2Fr0yfnpOqBjeIOFBu5jIM_ZFrw/edit?tab=t.0#heading=h.ayk2ggtcj1pz